:rotating_light: TEST MINIMUM DEPENDENCIES: failure

The run `25307864493` of the workflow TEST MINIMUM DEPENDENCIES failed.

You can view the report here: https://github.com/nilearn/nilearn/actions/runs/25307864493

Pinging @nilearn/core

From [the logs (L14905)](https://github.com/nilearn/nilearn/actions/runs/25307864493/job/74187744777#step:6:14906) :
```
________ ERROR at teardown of test_check_niimg_user_expand[~/test.nii1] ________
```

[L15087](https://github.com/nilearn/nilearn/actions/runs/25307864493/job/74187744777#step:6:15088) :
```
E           OSError: Expected 1176 bytes, got 0 bytes from /Users/runner/test.nii
E            - could the file be damaged?
```

I doubt this reproduces systematically ....

--- a/.github/workflows/testing_minimum.yml
+++ b/.github/workflows/testing_minimum.yml
@@ -12,6 +12,7 @@
 name: test minimal version of each dependency
 
 on:
+    pull_request:
     schedule:
         # Uses the cron schedule for github actions
         # https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule

--- a/.github/workflows/testing_minimum.yml
+++ b/.github/workflows/testing_minimum.yml
@@ -12,7 +12,6 @@
 name: test minimal version of each dependency
 
 on:
-    pull_request:
     schedule:
         # Uses the cron schedule for github actions
         # https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule

--- a/.github/workflows/testing_minimum.yml
+++ b/.github/workflows/testing_minimum.yml
@@ -78,13 +78,7 @@ jobs:
             continue-on-error: true
             run: |
                 pytest -m "not flaky and not single_process" --pyargs nilearn --report=report.html --numprocesses auto
-                pytest -m "not flaky and single_process" --pyargs nilearn --report=report.html
-
-        -   name: Generate plots tests timing
-            if: success() || failure()
-            run: |
-                pip install tox
-                tox run -e plot_test_timing
+                pytest -m "not flaky and single_process" --pyargs nilearn
 
         -   name: Upload test report
             if: success() || failure()
@@ -92,13 +86,6 @@ jobs:
             with:
                 name: ${{ matrix.os }}_${{ matrix.min_dep }}_report.html
                 path: report.html
-
-        -   name: Upload test timings
-            if: success() || failure()
-            uses: actions/upload-artifact@v7
-            with:
-                name: ${{ matrix.os }}_${{ matrix.min_dep }}_pytest_output
-                path: results/pytest_output
 
         -   name: Create issue
             if: ${{ steps.test_minimum.outcome != 'success' }}

--- a/.github/workflows/testing_minimum.yml
+++ b/.github/workflows/testing_minimum.yml
@@ -1,5 +1,5 @@
 ---
-# This workflow is triggered when a new commit is pushed to the main branch (or when a pull request is merged) and is also automatically run once a month.
+# This workflow is automatically run once a month.
 #
 # Checks that installing the minimum version of a given dependency of Nilearn
 # along with the latest version of all the other dependencies leads to a successful run of all the tests.
@@ -77,7 +77,8 @@ jobs:
             # do not run flaky tests when testing minimum version of dependencies
             continue-on-error: true
             run: |
-                pytest -m "not flaky" --pyargs nilearn --report=report.html --numprocesses auto --csv results/pytest_output/pytest_output.csv
+                pytest -m "not flaky and not single_process" --pyargs nilearn --report=report.html --numprocesses auto
+                pytest -m "not flaky and single_process" --pyargs nilearn --report=report.html
 
         -   name: Generate plots tests timing
             if: success() || failure()

--- a/nilearn/image/tests/test_image.py
+++ b/nilearn/image/tests/test_image.py
@@ -2259,6 +2259,7 @@ def img_in_home_folder(img_3d_mni):
 
 
 @pytest.mark.thread_unsafe
+@pytest.mark.single_process
 @pytest.mark.parametrize(
     "filename", ["~/test.nii", r"~/test.nii", Path("~/test.nii")]
 )

